VectorRumble.Shared/Strings.Designer.cs

```diff
@@ -30,7 +30,7 @@ internal class Strings {
         internal static System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.Equals(null, resourceMan)) {
-                    System.Resources.ResourceManager temp = new System.Resources.ResourceManager("VectorRumble.Strings", typeof(Strings).Assembly);
+                    System.Resources.ResourceManager temp = new System.Resources.ResourceManager("VectorRumble.Strings", typeof(Strings).GetTypeInfo().Assembly);
```

Comment:
does this work on DesktopGL?

Reply:
Yes, hence the change. Builds on all platforms. Without this UWP would not compile.